woocommerce/woocommerce-gutenberg-products-block
# Shopper can place an order as logged in user

Fixes #5786

### Notes

This PR adds e2e tests to verify that a shopper can place an order while being logged in. 

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- tests/e2e/specs/shopper/checkout-place-order.test.js` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.
